client/my-sites/checkout/composite-checkout/composite-checkout.tsx

```diff
@@ -621,7 +614,7 @@ export default function CompositeCheckout( {
 		);
 	}
 
-	const updatedSiteId = isJetpackCheckout ? parseInt( responseCart.blog_id ) : siteId;
+	const updatedSiteId = isJetpackCheckout ? parseInt( String( responseCart.blog_id ) ) : siteId;
```

Comment:
I'm not sure how important this is (maybe some part of the inscrutable build process fixes it for us) but according to [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt) the `parseInt` function does not default to using 10 as the radix. As of [ES5](https://262.ecma-international.org/5.1/#sec-15.1.2.2) I think the only exception is that strings starting with `0x` are parsed as hex. If that happens here then something has gone very wrong, but what do you think about specifying 10 as the radix just to prevent the possibility of a silently mis-parsed string?

Reply:
Good point! I took your advice and put this commit in https://github.com/Automattic/wp-calypso/pull/52932